### ISSUE TITLE
Whatsapp 2.12.13

### ISSUE THREAD

**Issue by sampertini** [n00neimp0rtant/xCon-Issues#496]
Whatsapp detects installed jailbreak tweaks installed to modify itself (if any) and it closes automatically. It's been happening for several versions.

**Comment by Lunat1k** [comment]
Have you tried the latest beta on the main page? I use WhatsApp and it works fine.

**Comment by sampertini** [comment]
Yes, I did. I installed xCon Beta 5 and then also tried with Beta 1. With both, I installed "Couria", "Auki" or "Nuntius" tweaks, and when I open Whatsapp, it closes automatically. I then uninstall the tweaks, and Whatsapp runs just fine.
